lib/gnostic.go

```diff
@@ -37,6 +37,21 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+// UsageError is a response to invalid command-line inputs
+// and should trigger display of usage (help) information.
+type UsageError struct {
+	Message string
```

Comment:
Is exporting `Message` necessary? Should `Error()` be the only way of accessing the error string, or is direct `Message` access also acceptable?

Reply:
Thanks, I'm making this private.